### DISCUSSION TITLE
How to use index array to replace values present at indices with a number

### DISCUSSION THREAD

**Discussion by Prabakaran-Chandran** [josdejong/mathjs#2674] [Q&A] [answered]
I have indices in an array , say Index_array = [[0,1] , [1,2] , [2,3]] 

how to use this array for math.subset function . 

math.subset(array , math.index(Index_array) , 1) is not working 

Please help me

**Comment by josdejong** [comment, marked as answer]
Have you seen the docs and examples on how to use indexes, Prabakaran?

See: https://mathjs.org/docs/datatypes/matrices.html#getting-or-replacing-subsets
